### ISSUE TITLE
Magento 2.4.1 Commerce Edition - CRITICAL: Deprecated Funcitonality: array_key_exists()

### ISSUE THREAD

**Issue by complexthings** [shipperhq/module-shipper#81]
Magento Enterprise 2.4.1
PHP 7.4
Current Module Version 20.39.2

```
[2020-10-23 13:53:27] report.CRITICAL: Deprecated Functionality: array_key_exists(): Using array_key_exists() on objects is deprecated. Use isset() or property_exists() instead in /var/www/html/vend
or/shipperhq/module-shipper/src/Helper/Data.php on line 419 {"exception":"[object] (Exception(code: 0): Deprecated Functionality: array_key_exists(): Using array_key_exists() on objects is deprecate
d. Use isset() or property_exists() instead in /var/www/html/vendor/shipperhq/module-shipper/src/Helper/Data.php on line 419 at /var/www/html/vendor/magento/framework/App/ErrorHandler.php:61)"} []
```

**Comment by wsajosh** [comment]
Hi @complexthings Thanks for raising this. It shouldn't be causing any operational issues as its just a deprecation warning. I've created an internal task to fix this within the next few weeks nonetheless. I'll update here once its available

**Comment by wsajosh** [comment]
Hi @complexthings . This is fixed in the latest version - 20.39.5. Thanks for raising this.
